sky/utils/command_runner.py

```diff
@@ -134,17 +135,147 @@ class SshMode(enum.Enum):
     LOGIN = 2
 
 
-class SSHCommandRunner:
+class CommandRunner:
+    """Runner for commands to be executed on the cluster."""
+
+    def __init__(self, node_id: str, node: Any, **kwargs):
```

Comment:
What is the `node_id` field for?

Reply:
Oops, that is a remnant. Removed. Thanks!